.github/CODEOWNERS

```diff
@@ -2,24 +2,20 @@
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The repo admins team will be the default owners for everything in the repo.
```

Comment:
This looks like it now accidentally reverts https://github.com/prometheus-community/helm-charts/pull/44. Can you check main again and if make sure only your change is in this PR?